src/components/tabs/UIFrameworkTabs.astro

```diff
@@ -3,15 +3,17 @@ import Tabs from './Tabs';
 ---
 
 <Tabs client:visible sharedStore="ui-frameworks">
-	<Fragment slot="tab.preact">Preact</Fragment>
-	<Fragment slot="tab.react">React</Fragment>
-	<Fragment slot="tab.solid">Solid</Fragment>
-	<Fragment slot="tab.svelte">Svelte</Fragment>
-	<Fragment slot="tab.vue">Vue</Fragment>
+  <Fragment slot="tab.preact">Preact</Fragment>
+  <Fragment slot="tab.react">React</Fragment>
+  <Fragment slot="tab.solid">Solid</Fragment>
+  <Fragment slot="tab.svelte">Svelte</Fragment>
+  <Fragment slot="tab.vue">Vue</Fragment>
+  <Fragment slot="tab.z_lit">Lit</Fragment>
```

Comment:
```suggestion
  <Fragment slot="tab.zLit">Lit</Fragment>
```

Reply:
I was just looking into that lol!
Thanks!